src/commands/lambda/instrument.ts

```diff
@@ -506,7 +507,7 @@ export class InstrumentCommand extends Command {
         this.context.stdout.write(
           `UpdateFunctionConfiguration -> ${config.functionARN}\n${JSON.stringify(
             config.updateFunctionConfigurationCommandInput,
-            undefined,
+            maskUpdateFunctionConfigurationCommandInput(config.updateFunctionConfigurationCommandInput),
```

Comment:
I'm thinking of a better way of naming this, since it's quite verbose

Reply:
I just renamed it to `maskStringifiedEnvVar`. Let me know if you have any other suggestions!